.github/release-drafter.yml

```
@@ -1,19 +1,19 @@
-name-template: v$NEXT_PATCH_VERSION
-tag-template: v$NEXT_PATCH_VERSION
+name-template: 'v$NEXT_PATCH_VERSION'
+tag-template: 'v$NEXT_PATCH_VERSION'
 categories:
-  - title: 💡 New Features
-    label: feature
-  - title: 🔨 Breaking
-    label: breaking
-  - title: 🚀 Efficiency Improvement
-    label: efficiency
-  - title: 🐛 Bug Fixes
-    label: fix
-  - title: 📖 Documentation
-    label: doc
-  - title: 🧰 Maintenance
-    label: maintenance
-tag-template: - $TITLE @$AUTHOR (#$NUMBER)
+  - title: '💡 New Features'
+    label: 'feature'
+  - title: '🔨 Breaking'
+    label: 'breaking'
+  - title: '🚀 Efficiency Improvement'
+    label: 'efficiency'
+  - title: '🐛 Bug Fixes'
+    label: 'fix'
+  - title: '📖 Documentation'
+    label: 'doc'
+  - title: '🧰 Maintenance'
+    label: 'maintenance'
+change-template: '- TITLE @AUTHOR (#$NUMBER)'
```

I guess template signs `$` are required.
```suggestion
change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
```